doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -161,6 +161,17 @@ up to the size of the DH prime B<p>.
 If B<OSSL_EXCHANGE_PARAM_PAD> is zero (the default) then no padding is
 performed.
 
+=item OSSL_SIGNATURE_PARAM_MD_NAME (UTF8 string type)
+
+Sets the name of the digest algorithm used for the input to the signature
+functions.
+
+=item OSSL_SIGNATURE_PARAM_MD_SIZE (size_t type)
+
+Sets the output size of the digest algorithm used for the input to the signature
+functions.
+Internal algorithms that support this parameter are DSA.
```

Comment:
`The internal algorithm that supports this...`

If there is only one, it should be singular.

Reply:
I actually wrote it this way on purpose because I knew we were going to add more in the future. But for now I've written it as you suggested.